README.md

```diff
@@ -7,32 +7,59 @@
 
 ## Run ownCloud Infinite Scale
 
-Please see [Getting Started](https://owncloud.dev/ocis/getting-started/)
+Please see [Basic Remote Setup](https://owncloud.dev/ocis/deployment/basic-remote-setup/) for a single node bare metal deployment like on a Raspberry Pi or learn how to [deploy to Kubernetes](https://owncloud.dev/ocis/deployment/kubernetes/).
+
+To build and run a local instance with demo users:
+
+```console
+# get the source
+git clone git@github.com:owncloud/ocis.git
+
+# enter the ocis dir
+cd ocis
+
+# generate assets
+make generate
+
+# build the binary
+make -C ocis build
+
+# initialize a minimal oCIS configuration
+./ocis/bin/ocis init
+
+# run with demo users
+IDM_CREATE_DEMO_USERS=true ./ocis/bin/ocis server
+```
+
+All batteries included: no external database, no external IDP needed!
 
 ## Documentation
 
-*   **Development Documentation**  
-Please see [Development Documentation - Getting Started](https://owncloud.dev/ocis/development/getting-started/)
+### Development Documentation
+Please see [Development Documentation - Getting Started](https://owncloud.dev/ocis/development/getting-started/) to get an overview of [Requirements](https://owncloud.dev/ocis/development/getting-started/#requirements), the [repository structure](https://owncloud.dev/ocis/development/getting-started/#repository-structure) and [other starting points](https://owncloud.dev/ocis/development/getting-started/#starting-points).
 
-*   **Admin Documentation**  
-Please see [Admin Documentation - Introduction to Infinite Scale](https://doc.owncloud.com/ocis/next/)
+### Admin Documentation
+Please see [Admin Documentation - Introduction to Infinite Scale](https://doc.owncloud.com/ocis/next/) to get started with running oCIS in production.
 
 ## Security
 
-If you find a security issue please contact [security@owncloud.com](mailto:security@owncloud.com) first
+If you find a security issue please contact [security@owncloud.com](mailto:security@owncloud.com) first.
 
 ## Contributing
 
-Please refer to our [Contribution Guidelines](CONTRIBUTING.md).
-
-## License
+We are _very_ happy that oCIS does not require a CLA as it is [Apache 2.0 licensed](LICENSE). We hope this will make it easier to contribute code. If you want to get in touch, most of the developers hang out in our [rocket chat channel](https://talk.owncloud.com/channel/infinitescale). There are other ways to contribute, like the [owwnCloud central forum](https://central.owncloud.org/) or [Transifex for ownCloud web](https://explore.transifex.com/owncloud-org/owncloud-web/).
```

Comment:
```suggestion
We are _very_ happy that oCIS does not require a CLA as it is [Apache 2.0 licensed](LICENSE). We hope this will make it easier to contribute code. If you want to get in touch, most of the developers hang out in our [rocket chat channel](https://talk.owncloud.com/channel/infinitescale). There are other ways to contribute, like the [ownCloud central forum](https://central.owncloud.org/) or [Transifex for ownCloud web](https://explore.transifex.com/owncloud-org/owncloud-web/).
```